setup.cfg

```diff
@@ -39,12 +39,6 @@ python_requires = >=3.9
 include_package_data = True
 
 install_requires =
```

Comment:
It seems like this repository is in an odd state of dependency management. 

- If the python code in this repository is intended to be a package then the setup.cfg should list the necessary dependencies. This is so someone that buids the code and then installs with a package manager (i.e. pip) will 
get all the dependencies they need installed as well. 
- If the python code in this repository isn't a package then the setup.cfg, __init__.py, etc. structure is unnecessary and can all be removed. In that case the python code is just a set of files that organized by directory.

Given that the code has previously been vended to PyPi as a package I suggest we continue to manage it as such.

I realize these lines look like they're duplicative of the environment-*.yml files that are in this repository but they do very different things. This install_requires in setup.cfg is a statement about compatibility for this package and might specify a range of versions that will work with this code. The actual pinned versions of the libraries in the environment files define specific deployment configurations for the Docker images. Those environments should always be versions compatible with the values in setup.cfg but they don't always need to be the same.

Reply:
Updated to keep dependencies also in setup.cfg since we are indeed hosting this as a pypi repo - TY!